cmc-domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/ClaimDataTest.java

```diff
@@ -26,9 +25,9 @@
     @Test
     public void shouldHaveNoValidationMessagesWhenInterestTypeIsNoInterest() {
         //given
-        ClaimData claimData = noInterest();
+        ClaimData claimData = SampleClaimData.noInterest();
         //when
-        Set<String> response = validate(claimData);
+        Set<String> response = BeanValidator.validate(claimData);
```

Comment:
No need to change this, static import was fine (here and everywhere else).

Reply:
reverted.

Reply:
It's on other files as well unfortunately.

Reply:
it happened with optimise import using tool. cleaning this.